homeassistant/components/media_player/depict.py

```diff
@@ -0,0 +1,95 @@
+"""
+Support for controlling what images are displayed on the Depict art frame.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.depict/
+"""
+
+from homeassistant.components.media_player import (
+    SUPPORT_TURN_OFF, SUPPORT_TURN_ON, SUPPORT_PLAY_MEDIA, MediaPlayerDevice)
+from homeassistant.const import (STATE_ON, STATE_OFF)
+from ..depict import CONF_NAME, DATA_DEPICT
+
+DEPENDENCIES = ['depict']
+
+SUPPORTED_FEATURES = SUPPORT_TURN_OFF \
+    | SUPPORT_TURN_ON \
+    | SUPPORT_PLAY_MEDIA
+
+ATTR_ORIENTATION = 'orientation'
+ATTR_RESOLUTION = 'resolution'
+MEDIA_TYPE_IMAGE = 'image'
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
```

Comment:
If there's no I/O here we can make it a coroutine and rename to `async_setup_platform `.